assembl/static/js/app/views/attachments.js

```diff
@@ -261,6 +270,54 @@ var AttachmentFileEditableView = AttachmentEditableView.extend({
 
 });
 
+
+/*
+  The view used for attachments in the idea panel when attachment is editable
+  ie. when the user has the permission to upload a file.
+ */
+var AttachmentFileEditableViewIdeaPanel = AttachmentFileEditableView.extend({
+  initialize: function(options){
+    //Save the attachment as soons as the document is saved
+    var doc = this.model.getDocument();
+    this.listenToOnce(doc, 'sync', this.onDocumentSave);
+    AttachmentFileEditableView.prototype.initialize.call(this, options);
+  },
+
+  modelEvents: {
+    'change': 'onChange',
+    'destroy': 'onDestroy'
+  },
+
+  onDocumentSave: function(documentModel, resp, options){
+    //Save the attachment model as well, as in the idea panel, there is no confirmation
+    //to save the attachment
+    this.model.save();
+  },  
+
+  /*
+    There is a limit of 1 attachment, so this *should* only be called once
+   */
+  onChange: function(e){
+    var domObject = $(".content-ideapanel");
+    this.$el.find(".embedded-image-preview").load(function() {
```

Comment:
Why not factorize this code with the code in checkContentHeight()? (or use pure CSS, see my other comment)